js/interface.js

```diff
@@ -1946,12 +1948,11 @@ var DynamicLists = (function() {
       $filterAccordionContainer.append($newPanel);
 
       if (['empty', 'notempty', 'between'].indexOf(data.logic) !== -1) {
-        $newPanel.find('.panel-title-text .value, #value-dash, #filter-value-type').addClass('hidden');
-        $newPanel.find('.panel-title-text .value, #value-dash, #filter-value').addClass('hidden');
+        $newPanel.find('.panel-title-text .value, #value-dash, #filter-value-type, #filter-value').addClass('hidden');
       }
 
       if (data.logic !== 'between') {
-        $newPanel.find('.panel-title-text .value, #value-dash, #logic-comparison').addClass('hidden');
+        $newPanel.find('#logic-comparison-' + data.id).addClass('hidden');
```

Comment:
You don't need the context of `$newPanel` if you're referring to an element of unique ID on the page. Please correct all other similar mistakes in this PR too.

```suggestion
        $('#logic-comparison-' + data.id).addClass('hidden');
```

Reply:
@tonytlwu I corrected everything about your comment in the pr